src/Pages/Home/Home.jsx

```diff
@@ -1,12 +1,23 @@
-import { Button } from "../../Components";
+import { useState, useEffect } from "react";
+import { Button, ArticleCard } from "../../Components";
 
 import style from "./Home.module.scss";
 
 const Home = () => {
+  const [msg, setmsg] = useState([]);
+  useEffect(() => {
+    fetch("/articles/getarticles.json")
+      .then((res) => res.json())
+      .then((jres) => setmsg(jres?.msg));
```

Comment:
Handle error as well. Manage a state for handling errors and catch any error in the fetch. Also any error if 

`jres.success === false`

must be handled properly and show proper error response using the error state. Ask UI/UX Team for error in fetching the article's design and implement the design.